Add a note on default value for maxDaysSinceUpdate

OmarShehata opened the pull request:
I linked to the file instead of saying what the default value is so we don't have the same thing in two places.

cesium-concierge commented:
@OmarShehata, thanks for the pull request! Maintainers, we have a signed CLA from @OmarShehata, so you can review this at any time.

I noticed that [CHANGES.md](https://github.com/AnalyticalGraphicsInc/cesium-concierge/blob/master/CHANGES.md) has not been updated. If this change updates the public API in any way, fixes a bug, or makes any non-trivial update, please add a bullet point to `CHANGES.md` and comment on this pull request so we know it was updated. For more info, see the [Pull Request Guidelines]( https://github.com/AnalyticalGraphicsInc/cesium/blob/master/CONTRIBUTING.md#pull-request-guidelines).

---

:sparkles: :sparkles: :sparkles:__I'm the Cesium Concierge and I'm here to test things.__ :sparkles: :sparkles: :sparkles:

OmarShehata commented:
I updated the readme based on your feedback, and also added a "Default" column for all variables. Required variables have no default, and variables that, when not set, will disable the check, will have a note in the default column saying "Disabled if not set".

ggetz commented:
Looks good, thanks for cleaning this up @OmarShehata !
